components/automate-ui/README.md

```diff
@@ -88,6 +81,9 @@ Reason: Later releases are release candidates; should only be using production-r
 
 Per https://github.com/chef/automate/pull/1867, future versions have made a breaking change so it breaks our code base.
 
+### Package zone.js: ^0.10.3
+
+Per: https://www.npmjs.com/package/zone.js?activeTab=readme, Starting with v0.11.1, A breaking change is introduced for legacy browsers such as IE11.  If we are no longer supporting IE 11, we are safe to upgrade, otherwise we can instead `import 'zone.js/dist/zone';`
```

Comment:
If we're comfortable with not supporting IE 11, we can update.  If not we can use a backwards compatible import

Reply:
Since Microsoft is just starting de-commissioning support for IE 11--with some products still supporting it throughout 2021--I am thinking we should probably be conservative and continue supporting it, too.

Reply:
We'll have to wait to upgrade zone.js to `0.11.1` until the release of `angular/core 11.1.0` otherwise we'll get an incompatible peer dependency warning.  Should be cleared up in the next release re: https://github.com/angular/angular/issues/39094#issuecomment-742227156